hw2/04-api.js

```diff
@@ -1,7 +1,22 @@
-/** Exercise 04 - API **/
+/** Exercise 04 - API * */
 
-const url = 'https://restcountries.com/v3.1/all';
+const url = 'https://restcountries.com/v2/all'
+const results = document.querySelector('#results')
 
-// Add your code here
+const getData = (myUrl) => {
+    fetch(myUrl)
+        .then((response) => {
+            return response.json()
+        })
+        .then((countries) => {
+            countries.forEach((country) => {
+                const countryItem = document.createElement('li')
 
-getData(url);
+                countryItem.innerText = `${country.name} - ${country.population.toLocaleString()}`
```

Comment:
nice use of localeString to get the sorting to work correctly

Reply:
toLocalString is used to format number with comma's. Sorting is working here because API version used is 2.0 and not 3.1